spark/src/main/kotlin/com/adevinta/spark/components/textfields/SparkTextField.kt

```diff
@@ -87,10 +93,10 @@ internal fun SparkTextField(
     placeholder: String?,
     helper: String?,
     counter: TextFieldCharacterCounter?,
-    leadingIcon: @Composable (() -> Unit)?, // Should we rename it to leadingContent?
-    trailingIcon: @Composable (() -> Unit)?, // Should we rename it to trailingContent?
-    isError: Boolean,
-    error: String?,
+    leadingContent: @Composable (() -> Unit)?,
+    trailingContent: @Composable (() -> Unit)?,
```

Comment:
Is there anything else than an Icon that could be passed? Maybe this can be restricted

Reply:
Yeah it can be a suffix / dropdown and any other plugin